src/core/session.webjs.core.ts

```diff
@@ -304,11 +304,17 @@ export class WhatsappSessionWebJSCore extends WhatsappSession {
   /**
    * Group methods
    */
-  public createGroup(request: CreateGroupRequest) {
+  public async createGroup(request: CreateGroupRequest) {
     const participantIds = request.participants.map(
       (participant) => participant.id,
     );
-    return this.whatsapp.createGroup(request.name, participantIds);
+    const groupInfo = await this.whatsapp.createGroup(request.name, participantIds);
+    return groupInfo;
```

Comment:
Isn't it the same? I believe nestjs resolve all promises by default.
But it can be useful for debugging purpose, I'm fine with that!